### DISCUSSION TITLE
Request to Add a Translation Feature in the GitHub App

### DISCUSSION THREAD

**Discussion by ju8sho** [community/community#137375] [Mobile] [answered]
Dear GitHub Development Team,

I hope this message finds you well. I am writing to suggest a feature that I believe would significantly enhance the GitHub experience for users worldwide.

As GitHub continues to grow globally, the language barrier has become a challenge for many developers. This barrier can sometimes hinder effective collaboration and understanding among developers from different linguistic backgrounds. To address this, I would like to propose the addition of a translation feature within the GitHub app.

This feature could allow users to translate text within repositories, issues, pull requests, and comments directly in the app. It would be immensely helpful for developers who are not fluent in English or those who are working with collaborators from different countries.

Thank you for considering this suggestion. I believe that adding a translation feature would make GitHub more accessible and inclusive for developers around the world.

**Comment by elisealix22** [comment, marked as answer]
Hi @ju8sho, thank you for the thoughtful request! Do you have a language in particular that you are hoping we support? We recently added [German and Korean](https://github.blog/changelog/2023-11-21-hallo-%ec%95%88%eb%85%95%ed%95%98%ec%84%b8%ec%9a%94-github-mobile-is-now-also-available-in-german-and-korean/) translations and are open to more suggestions as well!

**Reply by ju8sho** [reply]
Hi, thank you for your response! I would like to suggest adding an Uzbek translation feature for GitHub mobile. My goal is to enable users to translate various projects available in different languages into Uzbek within the app. This would greatly help developers and users from Uzbekistan access content more easily in their native language. Thank you for considering this request!

**Reply by ju8sho** [reply]
I apologize for the delay in my response, and thank you for your patience!

**Reply by TPS** [reply]
Possible to integrate Bing's full autotranslate mode?  Then everything's available!
